src/ignition/ignition.go

```diff
@@ -26,20 +30,34 @@ func NewIgnition() *ignition {
 }
 
 // ParseIgnitionFile reads an ignition config from a given path on disk
-func (i *ignition) ParseIgnitionFile(path string) (*config31types.Config, error) {
+func (i *ignition) ParseIgnitionFile(path string) (*types.Config, error) {
 	configBytes, err := ioutil.ReadFile(path)
 	if err != nil {
 		return nil, errors.Wrapf(err, "error reading file %s", path)
 	}
-	config, _, err := config31.Parse(configBytes)
+	config, err := i.parseToLatest(configBytes)
 	if err != nil {
-		return nil, errors.Wrap(err, "error parsing ignition")
+		return nil, err
 	}
-	return &config, nil
+	return config, nil
+}
+
+// ParseIgnitionFile reads an ignition config from a given path on disk
+func (i *ignition) parseToLatest(content []byte) (*types.Config, error) {
```

Comment:
Why not use the same code as the service (where we force 3.2...)?
I think you can actually use the same code as a library...

Reply:
it is the same code but it gives us an option just to change import and not go through all variables

Reply:
so it is just naming thing

Reply:
Ok